lib/update-lockfile.js

```diff
@@ -53,7 +52,7 @@ module.exports = function updateLockfile (dependency, options) {
   if (exec('git status --porcelain').toString() === '') return
 
   // stage the updated lockfile
-  const ignoreOutput = info.ignoreOutput ? info.ignoreOutput : '2>/dev/null || true'
+  const ignoreOutput = options.ignoreOutput ? options.ignoreOutput : '2>/dev/null || true'
```

Comment:
Here I'm using the method for ignoring output defined in the ci-service, if its available.